Library/do/reg-models.do

```diff
@@ -9,65 +9,64 @@
 	The Lancet Infectious Diseases. 2016 Nov 30;16(11):1261-8.
 */
 
-    global 	graph_opts title(, justification(left) color(black) span pos(11)) graphregion(color(white)) ylab(,angle(0) nogrid) xtit(,placement(left) justification(left)) yscale(noline) xscale(noline) legend(region(lc(none) fc(none)))
+  global 	graph_opts title(, justification(left) color(black) span pos(11)) graphregion(color(white)) ylab(,angle(0) nogrid) xtit(,placement(left) justification(left)) yscale(noline) xscale(noline) legend(region(lc(none) fc(none)))
 
-    use 	"https://github.com/worldbank/stata-visual-library/raw/master/Library/data/reg-models.dta" , clear
+  use 	"https://github.com/worldbank/stata-visual-library/raw/master/Library/data/reg-models.dta" , clear
 
-    cap mat drop theResults
-    local 	theLabels ""
-    local 	x = 15.5
+  cap mat drop theResults
+  local 	theLabels ""
+  local 	x = 15.5
 
-    qui foreach var of varlist ??_correct refer med_any med_class_any_6 med_class_any_16 {
+  qui foreach var of varlist ??_correct refer med_any med_class_any_6 med_class_any_16 {
 
-        local theLabel : var label `var'
-        local theLabels `"`theLabels' `x' "`theLabel'""'
-        local x = `x' - 2
-    
-        reg `var' facility_private i.case_code 
-        
-        mat a = r(table)
-        
-        local 	b 	= a[1,1]
-        local 	ll 	= a[5,1]
-		local 	ul	= a[6,1]
-		mat 	a 			= [`b',`ll',`ul',1]
-        mat 	rownames a 	= "`var'"
-        
-        logit `var' facility_private i.case_code 
-        margins , dydx(facility_private)
-        
-        mat 	b	= r(table)
-        
-        local 	b 	= b[1,1]
-        local 	ll 	= b[5,1]
-        local 	ul 	= b[6,1]
-        mat 	b 	= [`b',`ll',`ul',2]
-        mat rownames b = "`var'"
-        
-        mat theResults = nullmat(theResults) \ a \ b 
-        
-        }
-        
-    mat colnames theResults = "b" "ll" "ul" "type"
-    matlist theResults
+    local theLabel : var label `var'
+    local theLabels `"`theLabels' `x' "`theLabel'""'
+    local x = `x' - 2
 
-    clear
-    svmat theResults , names(col)
+    reg `var' facility_private i.case_code 
```

Comment:
Can we add a few comments explaining what this part of the code is doing? The do-file is a lot more complex than most, so would be good to explain the steps

Reply:
Included in 9cc0c5082b1fb334af9daae4886b7db81b479f4c